src/zcl_abapgit_environment.clas.abap

```diff
@@ -0,0 +1,36 @@
+CLASS zcl_abapgit_environment DEFINITION
+  PUBLIC
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    CLASS-METHODS is_cloud
+      RETURNING
+        VALUE(rv_cloud) TYPE abap_bool .
+  PROTECTED SECTION.
+
+    CLASS-DATA gv_cloud TYPE abap_bool VALUE abap_undefined ##NO_TEXT.
```

Comment:
According to the above comment I would also recommend to change the name of the static variable to gv_sap_cloud_plattform. One alternative would be to use one variable with a kind of code line which indicated the enviroment like onPrem, SAP CP, public Cloud, private Cloud, .... according to the code list of the underlined method in class CL_COS_UTILITIES. What do you mean?

Reply:
for now we only need to know if its running on ABAPinSAPCP, we can add more options if needed, but as long as its not needed then I think it should be left out

Reply:
ok, then keep it simple :-)